.github/workflows/buid_tester.yml

```diff
@@ -0,0 +1,45 @@
+name: Build and Push flashlight tester image
+on:
+  push:
+    branches: [main]
```

Comment:
Can we update this for running on pull requests (since it's not merged it'll only run on this PR) so we can see if the pipeline is working as expected, then we can revert the commit change

Reply:
Forget it was already executed and worked

Reply:
https://github.com/getlantern/flashlight/actions/runs/12279823034

Reply:
@WendelHime yeah, I did test it before. I didn't want it to run on EVERY PR though